modules/util/JSUtil.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Gets object's values.
+ * @param {Object} object
+ * @return {Array<Object>}
+ */
+export function getValues(object) {
```

Comment:
We can depend on a shim: https://github.com/es-shims/Object.values just like we do for `for..of`.